src/m_menu.c

```diff
@@ -4034,14 +4049,15 @@ setup_menu_t gen_settings2[] = { // General Settings screen2
   {"Solid Status Bar Background", S_YESNO, m_null, M_X,
    M_Y + general_solidbackground*M_SPC, {"st_solidbackground"}},
 
-  {"", S_SKIP, m_null, M_X, M_Y + general_stub2*M_SPC},
+  {"Level Brightness", S_THERMO, m_null, M_X_THRM,
```

Comment:
This is a bit more than a "Display Option", right? Isn't this plain cheating in certain (very dark) maps?

Reply:
Yes, I disabled it in `strictmode`. We can move it to another menu group, maybe QOL?

Reply:
Yes, please. I see it rather in the "disable power-up flashes" category.